_posts/2020-09-01-mathieu_tanneau.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Mathieu Tanneau | Design and implementation of the interior-point solver Tulip"
+date:   2020-09-01
+categories: [jump-dev]
+---
+
+Mathieu Tanneau, a Ph.D. student at the Polytechnique Montréal, gave his talk
+that was accepted for JuMP-dev 2020 on the design and implementation of the
+interior-point solver [Tulip.jl](https://github.com/ds4dm/Tulip.jl) during
+August 2020's JuMP monthly developer call.
```

Comment:
Maybe add a link to https://github.com/mtanneau/jump-dev_2020
Cc @mtanneau

Reply:
Something like this would be enough:
```
Talk material is available [here](https://github.com/mtanneau/jump-dev_2020).
```